Packages/Tracking Preview/Input Action Hands/Leap Hand Inputs.inputactions

```diff
@@ -0,0 +1,274 @@
+                    "initialStateCheck": false
+                }
+            ],
+            "bindings": [
+                {
+                    "name": "",
+                    "id": "585ccf75-7403-46a8-972e-9a467b7a4374",
+                    "path": "<Leap Hand>{RightHand}/Palm Position",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "",
+                    "action": "Palm Position",
+                    "isComposite": false,
+                    "isPartOfComposite": false
+                },
+                {
+                    "name": "",
+                    "id": "3ae306a6-fc6b-4969-8107-1b66e96b883d",
+                    "path": "<Leap Hand>{RightHand}/Index Tip Position",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "",
+                    "action": "Index Tip Position",
+                    "isComposite": false,
+                    "isPartOfComposite": false
+                },
+                {
+                    "name": "",
+                    "id": "a5f59700-fe6e-4234-afff-5ac040d8cbe8",
+                    "path": "<Leap Hand>{RightHand}/Is Grabbing",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "",
+                    "action": "Grabbing",
+                    "isComposite": false,
+                    "isPartOfComposite": false
+                },
+                {
+                    "name": "",
+                    "id": "eb32d85c-2d24-4d70-a4be-ca9871e8a135",
+                    "path": "<Leap Hand>{RightHand}/Is Pinching",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "",
+                    "action": "Pinching",
+                    "isComposite": false,
+                    "isPartOfComposite": false
+                },
+                {
+                    "name": "",
+                    "id": "da824632-bfe6-440f-8d05-c65cf3f306a4",
+                    "path": "<Leap Hand>{RightHand}/Palm Direction",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "",
+                    "action": "Palm Direction",
+                    "isComposite": false,
+                    "isPartOfComposite": false
+                },
+                {
+                    "name": "",
+                    "id": "b48d1eb2-bd59-47ea-8466-e84ed390907a",
+                    "path": "<Leap Hand>{RightHand}/Tracked",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "",
+                    "action": "Tracked",
+                    "isComposite": false,
+                    "isPartOfComposite": false
+                }
+            ]
+        }
+    ],
+    "controlSchemes": [
+        {
+            "name": "New control scheme",
```

Comment:
Should this be called "New control scheme"?

Reply:
This file is auto-generated, but i'm happy to manually change the name :)